samples/affinidi-nextjs-nextauthjs/src/components/issuance/CredentialsRevocationTable.tsx

```diff
@@ -0,0 +1,182 @@
+import {
+  ChangeCredentialStatusInputChangeReasonEnum,
+  FlowData,
+  ListIssuanceRecordResponse,
+} from '@affinidi-tdk/credential-issuance-client';
+import React, { useState } from 'react';
+import Button from '../core/Button';
+import { UseQueryResult } from '@tanstack/react-query';
+
+interface PaginatedCredentialsTableProps {
+  handleRevoke: (
+    issuanceConfigurationId: string,
+    issuanceRecordId: string,
+    changeReason: string,
+    flowDataRecordsQuery: UseQueryResult<ListIssuanceRecordResponse, Error>
+  ) => Promise<FlowData>;
+  flowDataRecordsQuery: UseQueryResult<ListIssuanceRecordResponse, Error>;
+}
+
+const CredentialsRevocationTable: React.FC<PaginatedCredentialsTableProps> = ({
+  handleRevoke,
+  flowDataRecordsQuery,
+}) => {
+  const [isModalOpen, setIsModalOpen] = useState(false); // Modal visibility state
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasError, setHasError] = useState<string>();
+  const [selectedReason, setSelectedReason] = useState<
+    ChangeCredentialStatusInputChangeReasonEnum | ''
+  >('');
+  const [selectedFlowDataRecord, setSelectedFlowDataRecord] =
+    useState<FlowData>();
+
+  const userFriendlyFormat = (dateTime: string) => {
+    const date = new Date(dateTime);
+    return date.toLocaleString('en-US', {
+      weekday: 'long', // e.g., "Wednesday"
+      year: 'numeric', // e.g., "2024"
+      month: 'long', // e.g., "December"
+      day: 'numeric', // e.g., "18"
+      hour: 'numeric', // e.g., "11 AM"
+      minute: '2-digit', // e.g., "23"
+    });
+  };
```

Comment:
Do we need a user friendly date? This app is meant for developers, and they may find a simple `date.toISOString()` more useful as it includes timezone.

Reply:
was a request from Digvijay